assets/main.js

```diff
@@ -579,10 +579,32 @@ async function createVolume() {
 	const block_compressed = document.getElementById("block_compressed").checked;
 	const block_data = document.getElementById("block_data").files[0];
 
-	if (!isValidName(block_name)) {
```

Comment:
Thanks for refining the error message. Now, I'd appreciate if the `deployUnikernel()` as well gets such nice informative error messages, and best have both share the code....

So, could "isValidName" return the error message?